### DISCUSSION TITLE
120Hz option?

### DISCUSSION THREAD

**Discussion by DennisLofgren** [seerge/g-helper#3142] [Ideas]
I have an ROG Zephyrus G14 GA402NU and I would really appreciate a 120Hz mode just to keep consistency between my computers and other devices. The options I have are Auto, 60Hz and 165Hz+OD. There is space in the UI for another button too! 

If I could set it to 60 on battery and 120 when plugged in would be great too.

Thanks for the awesome work on this project!

**Comment by seerge** [comment]
@DennisLofgren hello, G-Helper shows options already available in windows (as it uses it's API to change them). Most likely you don't have 120Hz option in windows display settings in a first place, right ? :)

**Reply by DennisLofgren** [reply]
Very fast response! 

I checked in windows display settings and these are the options I have available:
![image](https://github.com/user-attachments/assets/5d18ef5a-9d69-4682-83f8-41bac8320b87)

I don't really know what dynamic 60 or 120 means.

**Reply by seerge** [reply]
@DennisLofgren Dynamic refresh rate is controlled by AMD driver - you can't set via windows API. If you want to use it, just select it in windows settings and make sure you don't have select `Auto` in G-Helper, then it won't interfere.
